internal/kafka/internal/config/dataplane_cluster_config.go

```diff
@@ -77,6 +77,20 @@ const (
 	NoScaling string = "none"
 )
 
+// constants for operators installation through OpenShift Lifecycle Manager (OLM)
+// in `standalone` cluster provider type
+const (
+	defaultStrimziOperatorOLMPackageName             = "kas-strimzi-bundle"
+	defaultStrimziOperatorOLMSubscriptionChannelName = "alpha"
+	defaultStrimziOperatorIndexImage                 = "quay.io/osd-addons/rhosak-strimzi-operator-bundle-index:v4.9-v0.1.5-2"
+	defaultStrimziOperatorSubscriptionConfigFile     = "config/strimzi-operator-subscription-spec-config.yaml"
+
+	defaultFleetShardOperatorOLMPackageName             = "kas-fleetshard-operator"
+	defaultFleetShardOperatorOLMSubscriptionChannelName = "alpha"
+	defaultFleetShardOperatorIndexImage                 = "quay.io/osd-addons/rhosak-fleetshard-operator-bundle-index:v4.9-v1.0.7-1"
```

Comment:
this image requires the pull secret

Reply:
One can build their own image following https://github.com/bf2fc6cc711aee1a0c2a/kas-installer/wiki/Deploying-kas-fleetshard-from-Source